ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/TrialTreeParserTest.java

```diff
@@ -76,6 +78,16 @@ public void testMiscParse() {
         testParse(MISC_TESTCASES, Node.class);
     }
 
+    @Test(expectedExceptions = TreeParserException.class)
+    public void testModuleDclnNameMainTest() throws TreeParserException {
+        testModuleDclnName(MODULE_DCLN_MAIN_TESTCASE);
+    }
+
+    @Test(expectedExceptions = TreeParserException.class)
+    public void testModuleDclnNameDoubleUnderscoreTest() throws TreeParserException {
+        testModuleDclnName(MODULE_DCLN_UNDERSCORE_TESTCASE);
+    }
```

Comment:
BTW, what about a test for the init function?

Reply:
Added in 5732df29120212bc9c5639fb1901b98d6172d290